### PR TITLE
DOM-task

### PR DESCRIPTION
**Pull request by mxnvkv**
### DOM-task

[Demo](https://mxnvkv.github.io/DOM-task/) |
[Code base](https://github.com/mxnvkv/DOM-task)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/mxnvkv/DOM task/script.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+function takeEl( selector ) {
+    return document.querySelector(selector);
+}
+
+// Creating needed elements
+
+var h1 = takeEl("h1");
+var main = takeEl(".main");
+var img = takeEl("img");
+var txt = takeEl("p");
```
Comment (Zakkarat) [inline submissions/mxnvkv/DOM task/script.js]: It is a bad practice using `var`, try using `let`, `const` instead.

File: submissions/mxnvkv/DOM task/script.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        `
+    },
+}
+
+function displayNavigation() {
+    nav.style.width = "200px";
+}
+
+function closeNavigation() {
+    nav.style.width = "0";
```
Comment (Zakkarat) [inline submissions/mxnvkv/DOM task/script.js]: There should not be any style in your js code.
JS for logic, CSS for styling

File: submissions/mxnvkv/DOM task/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    flex-direction: column;
+
+    width: auto;
+    height: calc(100vh - 100px);
+    padding: 50px;
+
+    background-color: #7D8CA3;
+}
+
+img {
```
Comment (Zakkarat) [inline submissions/mxnvkv/DOM task/css/styles.css]: Avoid styling by tag or id.
It's better to separate responsibilities.

File: submissions/mxnvkv/DOM task/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+--------------------------------------------------------------------
+********************************************************************
+
+                                Navigation
+
+********************************************************************
+--------------------------------------------------------------------
+*/
+
+nav {
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/css/styles.css]: Please, avoid styling by tag.

File: submissions/mxnvkv/DOM task/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/css/styles.css]: Please use one-style for your code. IMO you can delete this empty line. Also check yourself in following code.

File: submissions/mxnvkv/DOM task/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+    color: #B2B2B2;
+
+    font-size: 50px;
+}
+
+.cross:hover {
+    color: white;
+}
+
+ul {
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/css/styles.css]: :eyes: Avoid styling by tag.

File: submissions/mxnvkv/DOM task/index.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    </nav>
+    <header>
+        <div class="menu-wrapper">
+            <a href="#" class="menu"></a>
+        </div>
+        <h1></h1>
+    </header>
+    <div class="main">
+        <img src="">
+        <p></p>
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/index.html]: Why you're using empty `p` tag here?

File: submissions/mxnvkv/DOM task/index.html [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    </div>
+    <script src="script.js" charset="utf-8"></script>
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/index.html]: `charset` attribute is unnecessary. Where are you found this approach ?

File: submissions/mxnvkv/DOM task/script.js [hunk 9]
```diff
@@ -0,0 +1,132 @@
+// Creating object with cities
+
+var cities = {
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/script.js]: Using `const` for this variable will be better.

File: submissions/mxnvkv/DOM task/script.js [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+    nav.style.width = "200px";
+}
+
+function closeNavigation() {
+    nav.style.width = "0";
+}
+
+// Function that replaces current city's info to the clicked one
+
+function placeData( city ) {
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/script.js]: Please, try to use modern approach. `ES6` and `arrow function` looks like better.

File: submissions/mxnvkv/DOM task/script.js [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+var nav = takeEl("nav");
+var openMenu = takeEl(".menu");
+var closeMenu = takeEl(".cross");
+var ul = takeEl("ul");
+
+// Adding default informatio on the page
+
+placeData("barcelona");
+
+openMenu.addEventListener("click", () => {
```
Comment (o-msh) [inline submissions/mxnvkv/DOM task/script.js]: `openMenu.addEventListener("click", displayNavigation);`

**Pull request closed without merging**